### PR TITLE
Change azure pipelines reporter to support multiple results files

### PR DESCRIPTION
**Pull request by alexperovich**

### CODE REVIEW

File: src/Microsoft.DotNet.Helix/Sdk/tools/azure-pipelines/reporter/test_results_reader/__init__.py [hunk 1]
```diff
@@ -79,19 +79,24 @@ def add_logs(tr, log_list):
 def read_results(dir):
     # type: (str) -> Iterable[TestResult]
 
-    print "Searching '{}' for test results files".format(dir)
-
     log_files = list(get_log_files(os.path.join(get_env("HELIX_WORKITEM_ROOT"), "..")))
-
     log_list = construct_log_list(log_files)
 
+    print "Searching '{}' for test results files".format(dir)
```
Comment (MattGal) [inline src/Microsoft.DotNet.Helix/Sdk/tools/azure-pipelines/reporter/test_results_reader/__init__.py]: You should be doing this in python-3 style

**Comment by alexperovich** [comment]
@MattGal PTAL again. I fixed if for python 3

**Comment by JunTaoLuo** [comment]
Can you let me know which version of the Helix SDK contains this fix when it's published? Thanks.

**Comment by alexperovich** [comment]
Yea, i'll let you know when there is a new version available.

**Comment by JunTaoLuo** [comment]
Looks like 2.0.0-beta.19253.2 has the fix. I'm updating to that version.

**Comment by alexperovich** [comment]
Yea, that version has the change. The build was broken last night.

**Pull request merged**